kpet/data.py

```diff
@@ -457,6 +457,20 @@ def validate_host_type_regex(self):
                                                host_type_regex.pattern,
                                                host_types))
 
+    def validate_maintainers(self):
+        """
+        Check that every test case has a maintainer.
+        Raises:
+            schema.Invalid when a test case doesn't have any maintainers
+        """
+        for suite in self.suites or []:
+            for case in suite.cases or []:
+                if not case.maintainers and not suite.maintainers:
+                    raise Invalid("There has to be a maintainer for each "
+                                  "test, either at suite level or at case "
+                                  "level; in suite: {}\ncase: {}".
+                                  format(suite.name, case.name))
+
```

Comment:
Could you please put this into `kpet.data.Suite.__init__()` instead? It only concerns the test suites separately, we don't need the context of the whole database to check cases have maintainers, only the suite context.